tests/xtimer_hang: test crashes on startup

Building and executing the test on `native` let the test start and crashes just after printing 
```
...
If it stops before, the test failed.

RIOT/tests/xtimer_hang/../../Makefile.include:286: recipe for target 'term' failed
make: *** [term] Speicherzugriffsfehler (Speicherauszug erstellt)
```
however for @cgundogan the test works (gcc 5.3.0)
for me not (gcc 4.9.2) same for @authmillenon and @haukepetersen (both gcc 4.8.4)

--- a/tests/xtimer_hang/main.c
+++ b/tests/xtimer_hang/main.c
@@ -28,7 +28,7 @@
 
 #define TEST_SECONDS        (10U)
 #define TEST_TIME           (TEST_SECONDS * SEC_IN_USEC)
-#define STACKSIZE_TIMER     (1024)
+#define STACKSIZE_TIMER     (THREAD_STACKSIZE_DEFAULT)
 
 char stack_timer1[STACKSIZE_TIMER];
 char stack_timer2[STACKSIZE_TIMER];

